### ISSUE TITLE
Bug: Null coalescing operator in attribute removes curly braces

### ISSUE THREAD

**Issue by chasegiunta** [trivago/prettier-plugin-twig-melody#32]
```
<input type="text" name="fromName" value="{{ fromName ?? ' ' }}" />
```

```
<input type="text" name="fromName" value="fromName ?? ' '" />
```

**Comment by twbartel** [comment]
Will check, thanks!

**Comment by twbartel** [comment]
Hey Chase, I just published `v0.3.9` which should fix that issue. Thanks again for reporting!

**Comment by chasegiunta** [comment]
Awesome, works great. Thank you!
